lib/slot.ex

```diff
@@ -409,9 +416,22 @@ defmodule Tempus.Slot do
 
   defimpl Inspect do
     import Inspect.Algebra
+    # credo:disable-for-next-line
+    @fancy_inspect Application.get_env(:tempus, :fancy_inspect, false)
 
     def inspect(%Tempus.Slot{from: from, to: to}, opts) do
-      concat(["#Slot<", to_doc([from: from, to: to], opts), ">"])
+      opts.custom_options
+      |> Keyword.get(:fancy, @fancy_inspect)
```

Comment:
What's wrong with `if Keyword(opts.custom_options, :fancy, @fancy_inspect) do`? Too many pipes nope?

Also, maybe use `Keyword.get_lazy/3` and avoid the module attribute to support runtime configuration eventually, like `FANCY_KILLING_PERFORMANCE=true iex -S mix ...`. Anyway, thanks to throw emojis by default :bow:

Reply:
I am not sure how `Keyword.get_lazy/3` would improve performance here. The main goal was to eliminate `from:` and `to:` garbage from the inspected value.

I might have it as `#Spot<#{value}>`, the emoji is ruled out by default because it’s a notation change and I am to preserve the backward compatibility.

Reply:
Fixed with a clause with empty `custom_options`.

Reply:
@am-kantox A little bit late, but, shouldn't you use `compile_env` instead of `get_env` there? I though that elixir raises a warning now. :shrug:

Reply:
> Package published to https://hex.pm/packages/tempus/0.4.1